bundles/org.openhab.binding.electroluxair/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -181,5 +183,17 @@
 		<description>Ionizer Status</description>
 	</channel-type>
 
+	<channel-type id="uiLight">
+		<item-type>Switch</item-type>
+		<label>UI Light</label>
+		<description>Air Quality Light Status</description>
+	</channel-type>
+
+	<channel-type id="safetyLock">
+		<item-type>Switch</item-type>
+		<label>Safety Lock</label>
+		<description>Safety Lock Status</description>
+	</channel-type>
```

Comment:
Please update the I18N properties files since new strings have been added. See https://www.openhab.org/docs/developer/utils/i18n.html#generating-i18n-properties-file

Reply:
Fixed!